### DISCUSSION TITLE
Entry in other site reloads constantly?

### DISCUSSION THREAD

**Discussion by davidwebca** [craftcms/cms#15916] [Q&A] [answered]
Hi,

Just checking if this is normal. It seems an entry being modified in site A will provoke re-loading constantly in site B if you have them opened side-by-side. This is a bit jarring when copying some content over from one site to the other. 🤔 

Is it normal? Asking here before jumping to create an issue.
https://youtu.be/fwQhHmYg3FQ

**Comment by brandonkelly** [comment, marked as answer]
Yep, this is expected, because if you have any untranslated fields, their values get copied over to the other sites. So whenever you make a change to Site A, you’re also making changes to the other sites, indirectly.

**Reply by davidwebca** [reply]
Perfect, that's what I thought. But in this case it would be helpful if it did refresh only if there's data to copy - in my case there's none. Thanks for your answer!
